### PR TITLE
Prevent Duplicate Patient Record

### PR DESCRIPTION
**Pull request by MonishaSivanathan**
- Handled patient duplicate
- Patient exist it will throw error

### CODE REVIEW

File: controller/patient_controller.py [hunk 1]
```diff
@@ -22,9 +21,18 @@ def create_patient(pat: PatientModel):
         try:
             patient_id = get_unique_patient_id_json(pat.first_name, pat.last_name,
                                                     pat.date_of_birth)
+            print("patient_id", patient_id)
```
Comment (tamsud) [inline controller/patient_controller.py:24]: Print ?

File: controller/patient_controller.py [hunk 2]
```diff
@@ -74,6 +82,7 @@ def get_patient_by_id(patient_id: str):
             if patient:
                 logger.info(f"Patient Found: {patient_id}")
                 formatted_data = PatientClient.extract_patient_data(patient)
+                print("formatted_data", formatted_data)
```
Comment (tamsud) [inline controller/patient_controller.py:85]: Print?

File: controller/patient_controller.py [hunk 3]
```diff
@@ -22,9 +21,18 @@ def create_patient(pat: PatientModel):
...
             patient_id = get_unique_patient_id_json(pat.first_name, pat.last_name,
                                                     pat.date_of_birth)
+            print("patient_id", patient_id)
             # As this is open API, and we wont get Token here, so using default AIDBOX API.
             from aidbox.resource.patient import Patient
             #####
+
+            patient_id_update = get_md5([pat.first_name, pat.last_name, pat.date_of_birth])
+            existing_patient = PatientClient.get_patient_by_id(patient_id_update)
+            if type(existing_patient) == dict:
```
Comment (tamsud) [inline controller/patient_controller.py:31]: Chage that to wrapper call or default API call

**Review by tamsud — CHANGES_REQUESTED** [review]

File: controller/patient_controller.py (hunk 1 above)
Comment (MonishaSivanathan) [reply controller/patient_controller.py]: changed

File: controller/patient_controller.py (hunk 2 above)
Comment (MonishaSivanathan) [reply controller/patient_controller.py]: changed

File: controller/patient_controller.py (hunk 3 above)
Comment (MonishaSivanathan) [reply controller/patient_controller.py]: changed

**Review by tamsud — APPROVED** [review]

**Pull request merged**